discovery-provider/src/tasks/entity_manager/playlist.py

```diff
@@ -136,6 +138,17 @@ def validate_playlist_tx(params: ManageEntityParameters):
             raise Exception(
                 f"Cannot update playlist {playlist_id} that does not belong to user {user_id}"
             )
+    if params.action == Action.CREATE or params.action == Action.UPDATE:
+        playlist_metadata = params.metadata.get(params.metadata_cid)
+        if playlist_metadata:
+            playlist_description = playlist_metadata.get("description")
+            if playlist_description and len(playlist_description) > CHARACTER_LIMIT_PLAYLIST_DESCRIPTION:
+                raise Exception(f"Playlist {playlist_id} description exceeds character limit {CHARACTER_LIMIT_PLAYLIST_DESCRIPTION}")
+        if playlist_id in params.existing_records[EntityType.PLAYLIST]:
+            playlist: Playlist = params.existing_records[EntityType.PLAYLIST][playlist_id]
+            playlist_tracks = playlist.playlist_contents["track_ids"]
+            if (len(playlist_tracks) >= PLAYLIST_TRACK_LIMIT):
```

Comment:
i think it should actually check playlist_metadata here right? cuz the existing record will definitely be within limits we want to check the incoming record